recipes/c-ares/all/conandata.yml

```diff
@@ -1,29 +1,47 @@
 sources:
+  "1.19.0":
+    url: "https://github.com/c-ares/c-ares/releases/download/cares-1_19_0/c-ares-1.19.0.tar.gz"
+    sha256: "bfceba37e23fd531293829002cac0401ef49a6dc55923f7f92236585b7ad1dd3"
   "1.18.1":
     url: "https://github.com/c-ares/c-ares/releases/download/cares-1_18_1/c-ares-1.18.1.tar.gz"
     sha256: "1a7d52a8a84a9fbffb1be9133c0f6e17217d91ea5a6fa61f6b4729cda78ebbcf"
   "1.17.2":
-    sha256: 444b7dbf33cb6e8a764d069e3485cbaaa28386d60eace6988275c3cb9534d300
-    url: https://github.com/c-ares/c-ares/archive/refs/tags/cares-1_17_2.tar.gz
+    url: "https://github.com/c-ares/c-ares/releases/download/cares-1_17_2/c-ares-1.17.2.tar.gz"
+    sha256: "4803c844ce20ce510ef0eb83f8ea41fa24ecaae9d280c468c582d2bb25b3913d"
   "1.17.1":
-    sha256: 61f7cf09605f5e38d4828f82d0e2ddb9de8e355ecfd6819b740691c644583b8f
-    url: https://github.com/c-ares/c-ares/archive/cares-1_17_1.tar.gz
+    url: "https://github.com/c-ares/c-ares/releases/download/cares-1_17_1/c-ares-1.17.1.tar.gz"
+    sha256: "d73dd0f6de824afd407ce10750ea081af47eba52b8a6cb307d220131ad93fc40"
   "1.16.1":
-    sha256: 870962cc8f6b352303c404ce848e2ea1f1072f3c0a940042209a72179511c08c
-    url: https://github.com/c-ares/c-ares/archive/cares-1_16_1.tar.gz
+    url: "https://github.com/c-ares/c-ares/releases/download/cares-1_16_0/c-ares-1.16.0.tar.gz"
+    sha256: "de058ad7c128156e2db6dc98b8a359924d6f210a1b99dd36ba15c8f839a83a89"
```

Comment:
should be https://github.com/c-ares/c-ares/releases/download/cares-1_16_1/c-ares-1.16.1.tar.gz ;)

Reply:
@SpaceIm 
Sorry for my miss.
Fixed.